engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/RulesSet.scala

```diff
@@ -35,8 +34,10 @@ class RulesSet(rules: List[OAuth2ConfigRule], allCategories: List[String]) {
 }
 
 object RulesSet {
-  def getOnlyMatchingRules(roles: List[String], rules: List[OAuth2ConfigRule], allCategories: List[String]): RulesSet = {
+  def getOnlyMatchingRules(roles: List[String], rules: List[ConfigRule], allCategories: List[String]): RulesSet = {
     val filtered = rules.filter(rule => roles.map(_.toLowerCase).contains(rule.role.toLowerCase))
     new RulesSet(filtered, allCategories)
   }
+
+  def apply(rules: List[ConfigRule], allCategories: List[String]): RulesSet = new RulesSet(rules, allCategories)
```

Comment:
Is it needed? `getOnlyMatchingRules` is not the only usage? There u use constructor.

Reply:
You'r right - removed.